### PR TITLE
Move node management related guides from Core docs - Closes #561

### LINKED ISSUE #561: Move guides concerning node management into the SDK docs
## Which content is outdated?
The guides in the Core docs concerning node management.

## What should be the new content?
The guides should be added to the SDK docs, as management of a node is basically the same for all apps built with the Lisk SDK.

The Core docs should be updated to refer to the SDK docs were adequate, and only cover the Lisk Core specific content inside the Lisk Core documentation.

### PR DESCRIPTION
**Pull request by chris529**
Closes #561

### CODE REVIEW

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
+** xref:guides/app-development/test.adoc[Test a transaction]
+** xref:guides/app-development/interact.adoc[Interact with the API]
+** xref:guides/app-development/broadcast.adoc[Broadcast a transaction]
+** xref:guides/app-development/frontend.adoc[Connect a frontend]
+** xref:guides/app-development/launch.adoc[Launch a blockchain application]
+* Node management
+** xref:guides/node-management/forging.adoc[Enable forging]
+** xref:guides/node-management/api-access.adoc[API access]
+** xref:guides/node-management/enable-ssl.adoc[Enable SSL]
+** xref:guides/node-management/logging.adoc[Logging]
```
Comment (Tschakki) [inline modules/ROOT/nav.adoc]:
```suggestion
** Application development
*** xref:guides/app-development/configuration.adoc[Configure a blockchain application]
*** xref:guides/app-development/customize.adoc[Create a custom transaction]
*** xref:guides/app-development/test.adoc[Test a transaction]
*** xref:guides/app-development/interact.adoc[Interact with the API]
*** xref:guides/app-development/broadcast.adoc[Broadcast a transaction]
*** xref:guides/app-development/frontend.adoc[Connect a frontend]
*** xref:guides/app-development/launch.adoc[Launch a blockchain application]
** Node management
*** xref:guides/node-management/forging.adoc[Enable forging]
*** xref:guides/node-management/api-access.adoc[API access]
*** xref:guides/node-management/enable-ssl.adoc[Enable SSL]
*** xref:guides/node-management/logging.adoc[Logging]
```

File: modules/ROOT/pages/guides/index.adoc [hunk 2]
```diff
@@ -1,33 +1,45 @@
-= Guides
+= Management guides
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/index.adoc]:
```suggestion
= Guides
```

File: modules/ROOT/pages/guides/index.adoc [hunk 3]
```diff
@@ -1,33 +1,45 @@
...
 Mona Bärenfänger <mona@lightcurve.io>
-:description: This section provides an overview of the main guides in chronological order, together with an example using the 'Hello World' App.
+:description: This section provides an overview of the application development and node management guides.
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/index.adoc]:
```suggestion
:description: This section provides an overview of the application development and node management guides for the Lisk SDK.
```

File: modules/ROOT/pages/guides/index.adoc [hunk 4]
```diff
@@ -1,33 +1,45 @@
...
-. Finally, the xref:{url_launch}[launch the app guide] describes the steps needed to move from a local devnet to a public blockchain network.
+. The xref:{url_broadcast}[broadcast a transaction guide] describes how to create a transaction object of a specific transaction type, and how to broadcast it to the network.
+. The xref:{url_frontend}[frontend guide] covers how to create, setup, and run a frontend or client app that interacts with a Lisk blockchain network.
+. The xref:{url_launch}[launch guide] describes how to publish a blockchain application, which will enable other users to join the network, and will turn your local 1-node Devnet into an actual blockchain network.
+. The xref:{url_interact}[interact guide] covers how to interact with a blockchain network powered by the Lisk SDK, communicate with the API of a node, including examples of common interactions, for the transactions, blocks and accounts.
+. The xref:{test}[test guide] explains how to write unit tests required for transaction verification, including a working example.
+
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/index.adoc]: Although we could list here all the development related guies, I would prefer to focus here only on the most important ones. So imho the section from the old guides index can be used here again, no change needed.

File: modules/ROOT/pages/guides/node-management/logging.adoc [hunk 5]
```diff
@@ -0,0 +1,121 @@
+= Activate logging
+Mona Bärenfänger <mona@lightcurve.io>
+:description: This page explains the log levels and associated messages which can be used for monitoring or debugging a node.
+:toc:
+:url_bunyan: https://github.com/trentm/node-bunyan
+:url_bunyan_docs: http://trentm.com/node-bunyan/bunyan.1.html
+:url_monitoring_logs: {v_core}@lisk-core::pages/monitoring.adoc#log_monitoring
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/node-management/logging.adoc]:
```suggestion
:url_monitoring_logs: {v_core}@lisk-core::monitoring.adoc#log_monitoring
```

File: modules/ROOT/pages/guides/node-management/logging.adoc [hunk 6]
```diff
@@ -0,0 +1,121 @@
+= Activate logging
+Mona Bärenfänger <mona@lightcurve.io>
+:description: This page explains the log levels and associated messages which can be used for monitoring or debugging a node.
+:toc:
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/node-management/logging.adoc:4]:
```suggestion
:toc:
:v_core: master
```

**Review by Tschakki — CHANGES_REQUESTED** [review]
Looking pretty good already! 

Please see my comments and also remember to git remove the empty index pages in the app development and node management directories.

File: modules/ROOT/pages/guides/index.adoc (hunk 4 above)
Comment (chris529) [reply modules/ROOT/pages/guides/index.adoc]: OK, agreed and now done :-)

**Review by Tschakki — CHANGES_REQUESTED** [review]
Looking good, but on the `guides/index.adoc` page, there was a little misunderstanding, I think its good to have an additional section for the node management related guides, like you had it before. 

I would imagine to have 2 sections in the guides index, one called `App development` and one called `Node management`, each followed by a short description what this section is.

Then this can be followed by the overview of the main guides for the app development, like it is now.
For the node management, you can add back the overview of the node management guides that you created before.

The `Hello world app` should be a subsection of the `App development` section.

**Review by Tschakki — CHANGES_REQUESTED** [review]
Please make sure, to catch all links to the guides, that need to be updated.

On the following pages, I spotted broken links on staging:
https://liskhq.github.io/lisk-docs/lisk-sdk/setup.html
https://liskhq.github.io/lisk-docs/lisk-sdk/index.html
https://liskhq.github.io/lisk-docs/lisk-sdk/tutorials/hello-world.html
https://liskhq.github.io/lisk-docs/lisk-sdk/tutorials/transport1.html
https://liskhq.github.io/lisk-docs/lisk-sdk/reference/config.html
https://liskhq.github.io/lisk-docs/lisk-sdk/reference/lisk-commander/lisk-core.html
https://liskhq.github.io/lisk-docs/lisk-sdk/reference/lisk-framework/index.html

**Comment by chris529** [comment]
Hi Mona, OK, they should all work now.. :-)

File: modules/ROOT/pages/guides/index.adoc [hunk 7]
```diff
@@ -1,23 +1,29 @@
...
 
 This section consists of concise, goal-oriented how-to guides that are designed in a manner to help the user solve specific issues.
 
-The guides are listed in chronological order, walking the user through the basic steps of developing a blockchain application with the Lisk SDK.
 
-== Main Guides
+
+== Application development
+
+The guides are listed in chronological order, walking the user through the basic steps of developing a blockchain application with the Lisk SDK.
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/index.adoc]:
```suggestion
The guides in the application development section are aimed towards developers, who want to built their own blockchain application with the Lisk SDK.

The guides are listed in chronological order, walking the user through the basic steps of developing a blockchain application by the example of the <<hello_world_app, Hello World app>>.

=== Guides overview (application development)
```

File: modules/ROOT/pages/guides/index.adoc [hunk 8]
```diff
@@ -70,3 +76,21 @@ The delegates will then add a new block to the blockchain, which will also be vi
 image::run_a_blockchain_10_secs.gif[Logs of a node]
 
 All remaining topics of the Hello World application are explained further in the dedicated Lisk SDK guides.
+
+
+
+== Node management
+
+The node management guides are designed to assist the user by covering all the relevant aspects, right from providing recommendations to monitoring and debugging a node.
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/index.adoc]:
```suggestion
The node management guides are aimed towards admins and delegates, who operate a node that runs with the Lisk SDK.
Each of these guides covers a specific topic of node management, gives examples, and explains the available options.
```

File: modules/ROOT/pages/guides/index.adoc [hunk 9]
```diff
@@ -70,3 +76,21 @@ The delegates will then add a new block to the blockchain, which will also be vi
...
+
+The most important sections pertaining to the node management are described below:
+
+. The xref:{url_api_access}[API access guide] covers the options as well as how to control the access to a node, including the configurable flags.
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/index.adoc]:
```suggestion
. The xref:{url_api_access}[API access guide] explains, how to manage the API access to a node.
```

**Review by Tschakki — CHANGES_REQUESTED** [review]

**Comment by chris529** [comment]
Good points Mona.. 👍

**Review by Tschakki — APPROVED** [review]
Thank you!

**Pull request merged**